var/spack/repos/builtin/packages/charmpp/package.py

```diff
@@ -187,9 +187,9 @@ def charmarch(self):
             ("linux", "ppc", "pami"): "pami-linux-ppc64le",
             ("linux", "ppc", "verbs"): "verbs-linux-ppc64le",
             ("linux", "arm", "netlrts"): "netlrts-linux-arm7",
-            ("linux", "arm", "multicore"): "multicore-arm7",
+            ("linux", "arm", "multicore"): "multicore-linux-arm7",
             ("linux", "aarch64", "netlrts"): "netlrts-linux-arm8",
-            ("linux", "aarch64", "multicore"): "multicore-arm8",
+            ("linux", "aarch64", "multicore"): "multicore-linux-arm8",
```

Comment:
Sorry for the spam, could you please move these down to the version 6.11 if/else block?

Reply:
Done. Thanks for the quick reviews.